Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -261,8 +271,20 @@
   categories:
   - UplinkDeception
 
-# 弾薬
+# Ammo
 
+- type: listing
+  id: UplinkHighCaliberBox
+  name: uplink-high-caliber-box-name
+  description: uplink-high-caliber-box-desc
+  icon: { sprite: /Textures/Objects/Weapons/Guns/Ammunition/Boxes/anti_materiel.rsi, state: base-b }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)